tests/installation/releasenotes_origin.pm

```diff
@@ -21,11 +21,15 @@ sub run {
     assert_screen('release-notes-button');
     send_key('ctrl-shift-alt-x');
     assert_screen('yast-xterm');
-    my $src = check_var('SCC_REGISTER', 'installation') ? "RPM" : "URL";
+    # To find out if we could retrieve release notes we try to fetch the overview page
+    my $src = (script_run("curl -o /dev/null  https://www.suse.com/releasenotes/index.html",
```

Comment:
not sure if the SUT (Software Under Test) can reach that, but it should depend on arch and product version:
`https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP4/release-notes-sles.rpm`

Reply:
Another VR: https://openqa.suse.de/tests/7142949

In normal conditions when we expect the release notes from an URL, then the SUT surely can reach that because its the top directory of where it gets the release notes from. If the SUT can't reach this URL, then also YaST won't get release notes from an URL, and the logic will then accept release notes from RPM. If the SUT can reach that, but for any reasons the release notes for the most recent release are not yet ready (as we are experiencing now) the logic will record a soft failure.